ui/spec_helper.rb

```diff
@@ -32,4 +34,6 @@
 
 Capybara.configure do |config|
   config.default_driver = Drivers::CapybaraDrivers.chosen_driver
```

Comment:
I was going to ask you - did you help joseph with this setup?  otherwise I'm going to understand better what the changes require, or downgrade capybara

Reply:
> @ebrett I'm not seeing the `id`s on the elements you've added using `govuk_button_link_to`, either locally or in the review app, which for me means the element is never considered loaded.
> 
> In the review app the `completed@` user needed to go through the new registration because it was so old, then it wanted to resume to a page that has since been deleted I think.
> 
> I'm not sure how that allowed the specs to pass 🤷🏼‍♂️, maybe I am looking at the wrong thing (it is Friday after all); but I can't get the UI suite to pass locally in docker on the signup specs.
> 
> Could we recreate https://ey-recovery-pr-302.london.cloudapps.digital/ to be doubly sure.

Can do - I didn't realize the database did not get re-seeded with each deploy - can we change that?  That would be basic assumption as the point of review apps are to review the current code and that usually needs fresh data.

Reply:
How about if I just go in and reseed the database (like I do locally) - that's probably why the profile management specs were failing.  We need a big red sign that says old data on the review apps if we're not going to update the automatically.